docs/en/api/S3-API.md

```diff
@@ -139,16 +139,16 @@ Server: Jetty(9.2.z-SNAPSHOT)
 $ curl -i -X GET http://localhost:39999/api/v1/s3/testbucket
 
 HTTP/1.1 200 OK
-Date: Tue, 18 Jun 2019 21:25:27 GMT
+Date: Wed, 22 Sep 2021 07:15:19 GMT
 Content-Type: application/xml
-Content-Length: 354
-Server: Jetty(9.2.z-SNAPSHOT)
+Content-Length: 306
+Server: Jetty(9.4.43.v20210629)
 
-<ListBucketResult><Name>/testbucket</Name><Prefix/><ContinuationToken/><NextContinuationToken/><KeyCount>1</KeyCount><MaxKeys>1000</MaxKeys><IsTruncated>false</IsTruncated><Contents><Key>testobject</Key><LastModified>2019-06-18T14:24:33.029Z</LastModified><ETag></ETag><Size>27040</Size><StorageClass>STANDARD</StorageClass></Contents></ListBucketResult>
+<ListBucketResult><Contents><LastModified>2021-09-22T15:14:40.754Z</LastModified><Key>testobject</Key><Size>27040</Size></Contents><KeyCount>1</KeyCount><MaxKeys>1000</MaxKeys><Delimiter>/</Delimiter><EncodingType>url</EncodingType><IsTruncated>false</IsTruncated><Name>testbucket</Name></ListBucketResult>
 ```
 
 #### Listing a bucket with multiple objects
-You can upload more files and use the `max-keys` and `continuation-token` as the [GET bucket request parameter](https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html). For example:
+You can upload more files and use the `max-keys` and `marker` as the [GET bucket request parameter](https://docs.aws.amazon.com/AmazonS3/latest/API/API_ListObjects.html). For example:
```

Comment:
please reference ListObjectsV2 here

Reply:
`marker` is implemented in `listBuckets` interface, but `continuation-token` is not implemented.
`ListObjectsV2` did not contains `marker`, only `API_ListObjects` contains.

Reply:
@ZacBlanco @HelloHorizon 
Maybe we can open a new `type-feature` to support `ListObjectsV2` ?

Reply:
I see. Yes, we should open an issue for this. Thanks for pointing it out.